simply/market.py

```diff
@@ -228,21 +246,28 @@ def create_csv(self, filename, headers):
             writer = csv.writer(f)
             writer.writerow(headers)
 
-    def get_grid_fee(self, match):
+    def get_grid_fee(self, match=None, bid_cluster=None, ask_cluster=None):
```

Comment:
raise warning if match and (bid_cluster or ask_cluster)
'Either pass "match" or "bid_cluster"  and "ask_cluster"'